src/libs/geotagging.c

```diff
@@ -1583,25 +1591,26 @@ static gboolean _timezone_key_pressed(GtkWidget *entry, GdkEventKey *event, dt_l
   {
     case GDK_KEY_Return:
     case GDK_KEY_KP_Enter:
+    case GDK_KEY_Tab:
       _timezone_save(self);
-      gtk_window_set_focus(GTK_WINDOW(dt_ui_main_window(darktable.gui->ui)), NULL);
-#ifdef HAVE_MAP
-      dt_lib_geotagging_t *d = (dt_lib_geotagging_t *)self->data;
-      if(d->map.view)
-        _refresh_track_list(self);
-#endif
       return TRUE;
     case GDK_KEY_Escape:
       gtk_window_set_focus(GTK_WINDOW(dt_ui_main_window(darktable.gui->ui)), NULL);
       return TRUE;
-    case GDK_KEY_Tab:
-      return TRUE;
-    default:
+    default: ;
+      dt_lib_geotagging_t *d = (dt_lib_geotagging_t *)self->data;
+      gtk_label_set_text(GTK_LABEL (d->timezone_changed), " *");
```

Comment:
```suggestion
    default:
      {
        dt_lib_geotagging_t *d = (dt_lib_geotagging_t *)self->data;
        gtk_label_set_text(GTK_LABEL (d->timezone_changed), " *");
      }
```

would imo look better than lone `;` but that's my 2c ;)

Reply:
This is cleaner indeed !
Please do the change if you don't mind.